password/password.cabal

```diff
@@ -106,22 +134,33 @@ test-suite password-tasty
   type:
       exitcode-stdio-1.0
   hs-source-dirs:
+      src
       test/tasty
   main-is:
       Spec.hs
   other-modules:
-      Argon2
-    , Bcrypt
+      Data.Password.Internal
     , Internal
-    , PBKDF2
-    , Scrypt
     , TestPolicy
     , Validate
     , Paths_password
+  if flag(argon2)
+    other-modules:
+      Argon2
+  if flag(bcrypt)
+    other-modules:
+      Bcrypt
+  if flag(pbkdf2)
+    other-modules:
+      PBKDF2
+  if flag(scrypt)
+    other-modules:
+      Scrypt
   ghc-options:
       -threaded -O2 -rtsopts -with-rtsopts=-N
   build-depends:
       base >=4.9 && <5
+    , base64
```

Comment:
Where did this suddenly come from?

Is this (together with `, template-haskell`) a remnant of some testing?